Index.html

```diff
@@ -49,6 +46,32 @@ <h3 class="main-content__title">Home</h3>
         officia nihil doloribus. Quo harum explicabo corrupti ullam animi
         blanditiis saepe accusantium iusto architecto repellendus.
       </p>
+      <p>
+        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Autem illo
+        dicta voluptates aliquam, totam illum nihil, nostrum quia et similique
+        earum accusamus deleniti odit quidem temporibus deserunt voluptatibus
+        debitis vitae, odio nisi ad distinctio fuga ducimus. Rem ipsam mollitia,
+        labore asperiores nam fugit, assumenda corrupti velit voluptate iure
+        debitis blanditiis?
+      </p>
     </div>
+    <footer class="footer">
+      <div class="footer__container">
+        <div class="footer__copy">
+          <p>&copy; 2020. Damian Jaraba</p>
+        </div>
+        <div class="footer__social-icons">
+          <i class="icon icon--twitter footer__icon footer__icon-twitter"></i>
```

Comment:
Two `--` intstead of only one, same for the other icons
```suggestion
          <i class="icon icon--twitter footer__icon footer__icon--twitter"></i>
```